nexus_constructor/component.py

```diff
@@ -96,7 +100,7 @@ def transforms_full_chain(self):
         Gets all transforms in the depends_on chain for this component
         :return: List of transforms
         """
-        transforms = []
+        transforms = TransformationsList(self)
```

Comment:
Thanks!

Reply:
Just patching a few holes in the test coverage then I'll promote this PR for review